src/adapter.ts

```diff
@@ -230,6 +230,7 @@ export class MongooseAdapter {
     if (this.isSynced) {
       const session = await this.getSession();
       if (!session.inTransaction()) {
+        await CasbinRule.createCollection()
```

Comment:
Hmm. Is this really necessary? We use this library in production and also test environments where we rebuild database constantly and we have never seen this to be an issue.

Reply:
Mongodb will automatic execute this, but I think explicitly defined is also ok.

Reply:
Hmm. I would probably then move it to connect-function if adding it is necessary.

I have not seen this as common practice, so it would be good to add a comment to this as well on why createCollection is added. However, I might be wrong.